object.go

```diff
@@ -133,6 +133,7 @@ var (
 	ErrObjectAlreadyExists  = errors.New("nats: an object already exists with that name")
 	ErrNameRequired         = errors.New("nats: name is required")
 	ErrNeeds262             = errors.New("nats: object-store requires at least server version 2.6.2")
+	ErrLinkNotAllowed       = errors.New("nats: a link is not allowed with an object")
```

Comment:
I'm not sure about this error message (lacks context), maybe something like this would be cleaner:
```go
ErrLinkNotAllowed       = errors.New("nats: link cannot be set when putting the object in bucket")
```

Reply:
Changing